README.md

```diff
@@ -41,6 +41,7 @@
 - Use [Yarn Classic](#yarn-classic)
 - Use [Yarn Modern](#yarn-modern)
 - Use [Yarn workspaces](#yarn-workspaces)
+- Use [Yarn Plug'n'Play](#yarn-plugnplay)
```

Comment:
Let's move this section above `Yarn workspaces`.

Reply:
🤦‍♂️ Yep, totally should have caught that!